tests/headers/convert-cpp-comment-to-rust.hpp

```diff
@@ -1,4 +1,4 @@
-
+// bindgen-flags: --no-layout-tests
```

Comment:
This seems also unfortunate, how does this fail? It seems like a rather harmless test-case.

Reply:
```
     Running tests\convert-cpp-comment-to-rust.rs (target\debug\deps\convert_cpp_comment_to_rust-7c320c96cf75b56b.exe)

running 1 test
test bindgen_test_layout_mbedtls_mpi ... FAILED

failures:

---- bindgen_test_layout_mbedtls_mpi stdout ----
thread 'bindgen_test_layout_mbedtls_mpi' panicked at 'assertion failed: `(left == right)`
  left: `16`,
 right: `24`: Size of: mbedtls_mpi', tests\convert-cpp-comment-to-rust.rs:22:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

which is

```rust
pub type mbedtls_mpi_uint = ::std::os::raw::c_uint;
/// \brief          MPI structure
#[repr(C)]
#[derive(Debug, Copy, Clone)]
pub struct mbedtls_mpi {
    ///<  integer sign
    pub s: ::std::os::raw::c_int,
    ///<  total # of limbs
    pub n: ::std::os::raw::c_ulong,
    ///<  pointer to limbs
    pub p: *mut mbedtls_mpi_uint,
}
assert_eq!(
        ::std::mem::size_of::<mbedtls_mpi>(),
        24usize,
        concat!("Size of: ", stringify!(mbedtls_mpi))
    );
```